lib/assets/javascripts/utils/expandCollapseAll.js

```diff
@@ -0,0 +1,47 @@
+import 'bootstrap-sass/assets/javascripts/bootstrap.min';
```

Comment:
only import the bootstrap plugin that you are going to use (e.g. ... javascripts/bootstrap/collapse.js